packages/fxa-settings/src/components/Settings/FlowSetupRecoveryPhoneConfirmCode/en.ftl

```diff
@@ -6,7 +6,7 @@ flow-setup-phone-confirm-code-heading = Enter verification code
 
 # $phoneNumber is a partially obfuscated phone number with only the last 4 digits showing (e.g., *** *** 1234)
 # span element applies formatting to ensure the number is always displayed left-to-right
-flow-setup-phone-confirm-code-instruction = A six-digit code was sent to <span>{ $phoneNumber }</span> by text message. This code expires after 5 minutes.
+flow-setup-phone-confirm-code-instruction = A 6-digit code was sent to <span>{ $phoneNumber }</span> by text message. This code expires after 5 minutes.
```

Comment:
@bcolsson Following your recommendation from another recent PR, I didn't update the string id for this change.